setup.py

```diff
@@ -153,6 +153,7 @@ def load_requirements(file_list: Optional[Union[str, List[str]]] = None) -> List
         # Legacy extra requirements
         "huggingface": load_requirements("requirements/huggingface.in"),
         "local-inference": all_doc_reqs,
+        "paddleocr": load_requirements("requirements/paddleocr.in"),
```

Comment:
we don't have a file named `paddleocr.in` right (rename extra-paddle.in to paddle.in ?)